### PR TITLE
Resource fix for the Browser Capabilities Detection topic

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) fixes https://github.com/magento/merchdocs/issues/1524

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/stores/security-browser-capabilities-detection.html

### CODE REVIEW

File: src/stores/security-browser-capabilities-detection.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Like most websites and applications on the Internet, Adobe Commerce and Magento
 - If the browser’s privacy settings disallow cookies, you can configure the system to automatically redirect them to the [Enable Cookies]({% link cms/pages-core.md %}) page, which explains how to make the recommended settings with most browsers.
 - If the browser’s privacy settings disallow JavaScript, you can configure the system to display the following message above the header of every page.
 
-For technical information, see [Supported browsers][1]{:target="_blank"} in the developer documentation.
+For technical information, see [Supported browsers]({{ site.devdocs_url }}/install-gde/system-requirements_browsers.html#supported-browsers) in the developer documentation.
```
Comment (leslietilling) [inline src/stores/security-browser-capabilities-detection.md:10]: The install guide is versioned on devdocs, to it also needs that part of the URL

```suggestion
For technical information, see [Supported browsers]({{ site.devdocs_url }}/guides/v{{ site.version }}/install-gde/system-requirements.html#supported-browsers) in the developer documentation.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
A tweak for URL

File: src/stores/security-browser-capabilities-detection.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/security-browser-capabilities-detection.md]: Hi @leslietilling , thank you for reviewing. I have implemented your suggestion, could you please approve this PR?

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**